ovirt/resource_ovirt_vm.go

```diff
@@ -42,12 +42,10 @@ func resourceOvirtVM() *schema.Resource {
 			"name": {
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
```

Comment:
This might break some users - was the ForceNew here was a mistake from first place or deliberate? same for other fields.

Reply:
ForceNew: true - This behavior instructs Terraform to first destroy and then recreate the resource if any of the attributes change in the configuration, as opposed to trying to update the existing resource.

It wasn't mistake  from first place. Because if ovirt provider can't update parameter that one possible way to get resource with requested by customer configuration - this is destroy VM and create it again.

Removing line `ForceNew: true,` allow us to prevent destroy resource when this parameter will be change. As result we able to manage parameters in the resourceOvirtVMUpdate function.

Reply:
From my perspective, I would recommend to remove `ForceNew: true` to allow users change the name of VM without recreation. I've to admitted that it was a legacy for simple implementation. As declarative definition, it might be not a breaking change, since the configurations of the new VM are identical to the original VM, except the ID of VM. what's your opinions? @rgolangh @pavel-z1 .

Reply:
Ack. I was wondering what is the experience of handling an existing infrastructure using this provider and this
look like a necessary addition.